samples/gui/lvgl/boards/native_posix_64.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_SDL_DISPLAY=y
+CONFIG_SDL_DISPLAY_DEV_NAME="DISPLAY"
+CONFIG_SDL_DISPLAY_X_RES=320
+CONFIG_SDL_DISPLAY_Y_RES=240
```

Comment:
I don't claim to understand the nature of the problem here, but if we're having to do the same hack in two otherwise unrelated samples for two distinct platforms, it's probably time to come up with a framework to manage this better.  That is, the native_posix platforms would declare CONFIG_NEED_YADAYADA_SDL_WORKAROUND and the SDL code would implement the DEV_NAME and RES value fallbacks so the samples and tests don't have to.

Reply:
@andyross actually `CONFIG_SDL_DISPLAY` is not need as it is set by default for native_posix targets if `CONFIG_DISPLAY` is enabled and in this particular case the resolution parameters are also not needed as they are the default values. So this file could be reduced to `CONFIG_SDL_DISPLAY_DEV_NAME="DISPLAY"`, we could opt to use `DISPLAY` as default value.